src/main/webapp/app/exam/manage/exercise-groups/exercise-groups.component.html

```diff
@@ -59,7 +59,12 @@ <h5 class="font-weight-bold mb-0">{{ exerciseGroup.title }}</h5>
                             <fa-icon class="d-xl-none" [icon]="faCheckDouble"></fa-icon>
                             <span class="d-none d-xl-inline">{{ 'artemisApp.quizExercise.home.importLabel' | artemisTranslate }}</span>
                         </a>
-                        <a *ngIf="course?.isAtLeastEditor" [routerLink]="[exerciseGroup.id, 'quiz-exercises', 'new']" class="btn btn-info btn-sm me-1 mb-1" style="max-height: 44%">
```

Comment:
Did you remove the style="max-height: 44%" on purpose?

Reply:
Ah I think that was accidentally removed during a rebase. Thanks for the hint, I re-added the style.